src/ravindra_J/coding_exam_2/SumOfInteger.java

```diff
@@ -0,0 +1,42 @@
+package ravindra_J.coding_exam_2;
+
+/*
+Coding Exam - 2 : 26th April 2021
+Program 1 : 
+WAP to do sum of all the integer from the given string
+Input: String str = "12h14i4w8sdc15"
+Output: 12+14+4+8+15 = 53
+package ravindra_J.coding_exam_2;
+ */
+public class SumOfInteger {
+	int sum, num;
+	int getSumOfInteger(String str){
+		String concateDigit = "0";
+		for (int j = 0; j < str.length(); j++) {
+			char ch = str.charAt(j);
+
+			if(Character.isDigit(ch)) {
+				concateDigit = concateDigit+ch;				
+				num = Integer.parseInt(String.valueOf(concateDigit));
```

Comment:
Not up to the mark, num = Integer.parseInt(String.valueOf(concateDigit)); you are writing this line in both if and else why ??? 
Declaration of string should be like String concateDigit = ""; not String concateDigit = "0";

in else you should write else if( concateDigit != "")

Reply:
Screenshot is different than code, attach proper screenshot.